Makefile

```diff
@@ -0,0 +1,11 @@
+up:
+	docker-compose up -d hotness
```

Comment:
Will this work on podman only system?

Reply:
Yes, but it requires version +3.

Articles about that:
- https://podman.io/blogs/2021/01/11/podman-compose.html
- https://www.redhat.com/sysadmin/podman-docker-compose

Reply:
I know about the version restriction, although I wasn't sure, if you can use `docker-compose` instead of `podman-compose`.